cli/src/pcluster/aws/ec2.py

```diff
@@ -34,6 +34,10 @@ class Ec2Client(Boto3Client):
     def __init__(self):
         super().__init__("ec2")
         self.additional_instance_types_data = {}
+        self.security_groups_cache = {}
+        Cache.add_cache(self.security_groups_cache)
+        self.subnets_cache = {}
+        Cache.add_cache(self.subnets_cache)
```

Comment:
I'd add a comment here too, describing why we are adding these two dict into the cache.

Reply:
minor: I'd define all the default first, then I'd add them to the cache.
```python

        self.security_groups_cache = {}
        self.subnets_cache = {}
        Cache.add_cache(self.security_groups_cache)
        Cache.add_cache(self.subnets_cache)
```

Reply:
Not valid anymore